types/ink-select-input/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for ink-select-input 2.0
+// Project: https://github.com/vadimdemedes/ink-select-input#readme
+// Definitions by: Łukasz Ostrowski <https://github.com/lukostry>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { Component, InkComponent } from 'ink';
+
+interface ItemOfSelectInput {
+    label: string;
+    value: any;
+}
+
+interface SelectInputProps<T extends ItemOfSelectInput = ItemOfSelectInput> {
+    focus?: boolean;
+    indicatorComponent?: InkComponent;
+    itemComponent?: InkComponent;
+    items?: ReadonlyArray<T>;
+    limit?: number;
```

Comment:
Need optional property `initialIndex` per https://github.com/vadimdemedes/ink-select-input#initialindex ?

Reply:
you're referring to README for version 3.0 (apparently was published couple days ago). this PR provided definitions for version 2.0, when there was no such property: https://github.com/vadimdemedes/ink-select-input/tree/v2.0.0